backend/src/service/base_layers.rs

```diff
@@ -0,0 +1,44 @@
+//! Service layer for base layers (map component)
```

Comment:
A bit unclear what `(map component)` means. Maybe something like the following? Although it generally seems a bit weird to put this here.
```suggestion
//! Service layer for base layers.
//!
//! Layers are a part of the map.
```